app/models/participant_declaration.rb

```diff
@@ -7,6 +7,7 @@ class ParticipantDeclaration < ApplicationRecord
   belongs_to :user
   belongs_to :participant_profile
   belongs_to :superseded_by, class_name: "ParticipantDeclaration", optional: true
+  belongs_to :delivery_partner, optional: true
```

Comment:
Do we need the opposite relationship on `delivery_partner` for later?

Reply:
Maybe. I don't see that happening for some time yet though. At the moment I don't see the use case for a delivery partner to see all their declarations.